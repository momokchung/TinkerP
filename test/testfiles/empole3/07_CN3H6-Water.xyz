    13 94_CN3H6-Water_1.00
     1  C     -2.482941    0.145568    0.115882   158     3     2     4
     2  N     -3.741265    0.545114    0.294521   156     8     1     7
     3  N     -2.223001   -1.065026   -0.362487   156     9    10     1
     4  N     -1.472124    0.952664    0.413850   156     1     5     6
     5  H     -0.525470    0.627374    0.269800   157     4
     6  H     -1.631807    1.874145    0.777488   157     4
     7  H     -3.949510    1.458240    0.655417   157     2
     8  H     -4.513272   -0.056723    0.072692   157     2
     9  H     -2.962356   -1.701051   -0.598467   157     3
    10  H     -1.259909   -1.346152   -0.489506   157     3
    11  O      0.710698   -0.868442   -0.337400    16    12    13
    12  H      1.258534   -0.770165   -1.122733    17    11
    13  H      1.283422   -1.322547    0.288943    17    11
