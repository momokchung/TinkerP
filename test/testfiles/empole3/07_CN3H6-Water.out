
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            Tinker  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 8.10.5   February 2023                      ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2023               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Intermolecular Energy :             -19.95124123 Kcal/mole

 Total Potential Energy :            -23.14607036 Kcal/mole

 Energy Component Breakdown :           Kcal/mole        Interactions

 Atomic Multipoles                   -23.14607036               54
